galpy/potential_src/Potential.py

```diff
@@ -1539,73 +1580,49 @@ def rtide(self,R,z,M=None,M_units=None):
         
         HISTORY:
             
-            2018-02-12 - Written - Webb (UofT)
+            2018-03-21 - Written - Webb (UofT)
             
         """
+        #DO WE NEED THIS? PHI IS WORKED INTO RFORCE AND R2DERIV?
```

Comment:
If the result is sensible when just evaluating the forces and derivatives at R,z,phi,t then I agree that we don't need to do the check, so you can remove it.